app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
```

Comment:
Vì sao chỗ này dùng toán tử `||=`?

Reply:
Dạ do chức năng của toán từ '||=' là gán giá trị nếu giá trị được gán chưa có giá trị(nil hoặc false), ứng dụng trong chỗ này là kiếm tra người dùng hiện tại có ở trong phiên làm việc không, nếu có sẽ trả về thông tin người dùng hiện tại, còn không thì sẽ gán biến `current_user` với thông tin người dùng mới đăng nhập, giúp tránh phải tải dữ liệu người dùng trong mỗi phiên